apps/ide/src/plugins/project-wizard/build/build.js

```diff
@@ -14,11 +14,20 @@
  * limitations under the License.
  */
 
+/**
+ * @file Build runner
+ * @since 1.0.0
+ * @author cimfalab@gmail.com
```

Comment:
Hmm, is there a reason to change e-mail? (kh5325.kim is not my GitHub account :))